src/PrestaShopBundle/Resources/views/Admin/Improve/International/Locations/Country/Blocks/form.html.twig

```diff
@@ -0,0 +1,182 @@
+          'Call prefix'|trans({}, 'Admin.International.Feature'),
+          'International call prefix, (e.g. 1 for United States).'|trans({}, 'Admin.International.Help'),
+          '',
+          '',
+          true
+        ) }}
+        <div class="col-sm">
+          {{ form_widget(countryForm.call_prefix) }}
+          {{ form_errors(countryForm.call_prefix) }}
+        </div>
+      </div>
+
+      {{ ps.form_group_row(countryForm.default_currency, {}, {
+        'label': 'Default currency'|trans({}, 'Admin.International.Feature')
+      }) }}
+
+      <div class="form-group row">
+        {{ ps.label_with_help(
+          'Zone'|trans({}, 'Admin.Global'),
+          ('Geographical region.'|trans({}, 'Admin.International.Help')),
+          '',
+          '',
+          true
+        ) }}
+        <div class="col-sm">
+          {{ form_widget(countryForm.zone) }}
+          {{ form_errors(countryForm.zone) }}
+        </div>
+      </div>
+
+      {{ ps.form_group_row(countryForm.need_zip_code, {}, {
+        'label': 'Does it need Zip/postal code?'|trans({}, 'Admin.International.Feature')
+      }) }}
+
+      {{ ps.form_group_row(countryForm.zip_code_format, {}, {
+        'label': 'Zip/postal code format'|trans({}, 'Admin.International.Feature'),
+        'help' : 'Indicate the format of the postal code: use L for a letter, N for a number, and C for the country\'s ISO 3166-1 alpha-2 code. For example, NNNNN for the United States, France, Poland and many other; LNNNNLLL for Argentina, etc. If you do not want PrestaShop to verify the postal code for this country, leave it blank.'|trans({}, 'Admin.International.Help')
+      }) }}
+
+      <div class="form-group row">
+        <label class="form-control-label">
+          <span class="text-danger">*</span>
+          {{ 'Address format'|trans({}, 'Admin.Global')}}
+        </label>
+        <div class="col-sm">
+          {{ form_widget(countryForm.address_format, {'attr': {'style': 'min-height: 200px'}}) }}
+          {{ form_errors(countryForm.address_format) }}
+        </div>
+        <div class="col-sm">
+          {{ adr.tabNav(displayValidFields) }}
+          {{ adr.tabContent(displayValidFields) }}
+        </div>
+      </div>
+
+      <div class="form-group row justify-content-end">
+        {{ include('@PrestaShop/Admin/Improve/International/Locations/Country/Blocks/format_actions.html.twig',
+          {
+            'countryForm': countryForm,
+            'displayValidFields': displayValidFields,
+            'addressLayout': addressLayout,
+            'defaultLayout': encodingDefaultLayout,
+            'encodingAddressLayout': encodingAddressLayout
+          }
+        ) }}
+      </div>
+
+      <div class="form-group row">
+        {{ ps.label_with_help(
+          'Active'|trans({}, 'Admin.Global'),
+          'Display this country to your customers (the selected country will always be displayed in the Back Office).'|trans({}, 'Admin.International.Help'),
```

Comment:
```suggestion
          'Display this country to your customers (the selected country will always be displayed in the back office).'|trans({}, 'Admin.International.Help'),
```

Reply:
I cant see any translation key with lower case 'back office'. is it ok to change it?

Reply:
Yes, we will lose the translation (at least, at the beginning) but we will gain consistency and harmonization. ;-)

Reply:
👍